### PR TITLE
Adding Swift + Vapor Tutorial on GAE Flex

### PR DESCRIPTION
**Pull request by mcdonamp**
Adding a tutorial on Swift and Vapor. Perfect is on its way.

### CODE REVIEW

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM ibmcom/swift-ubuntu:latest
+LABEL Description="Docker image for Swift + Vapor on GAE Flex."
```
Comment (JustinBeckwith) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: "on Google App Engine" (no need to say flex)

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Expose default port for GAE
+EXPOSE 8080
+
+# Copy sources
+RUN mkdir /root/VaporGAE
```
Comment (JustinBeckwith) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: I think you can boil all of this (before the build) down to:

    COPY . /app/

No need to mkdir, or pick and choose individual files to copy.  Creating an `app` directory in the container is more consistent with our runtimes too.

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ADD main.swift /root/VaporGAE
+ADD Package.swift /root/VaporGAE
+
+# Build the app
+RUN cd /root/VaporGAE && swift build
```
Comment (JustinBeckwith) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: Better to set`WORKDIR /app` here, then you can just have a `RUN swift build` step and a simpler `CMD` below.

File: tutorials/run-swift-vapor-on-google-cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
+---
+title: Vapor on Google App Engine Flex Tutorial
```
Comment (JustinBeckwith) [inline tutorials/run-swift-vapor-on-google-cloud/index.md]: No `Flex` just `App Engine`

File: tutorials/run-swift-vapor-on-google-cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+env: flex
+```
+
+1.  Run the following command to deploy your app (make take several minutes):
+
+```
+gcloud app deploy
+```
+
+1.  Visit `http://[YOUR_PROJECT_ID].appspot.com/hello` to see the deployed app.
```
Comment (JustinBeckwith) [inline tutorials/run-swift-vapor-on-google-cloud/index.md]: Just run `gcloud app browse` :)

Comment (mcdonamp) [inline tutorials/run-swift-vapor-on-google-cloud/index.md]: Got it. Also had to add a line about adding "/hello" to the end

File: tutorials/run-swift-vapor-on-google-cloud/index.md (hunk 4 above)
Comment (mcdonamp) [inline tutorials/run-swift-vapor-on-google-cloud/index.md]: Done

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile (hunk 3 above)
Comment (mcdonamp) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: Awesome. https://github.com/GoogleCloudPlatform/appengine-custom-runtimes-samples/blob/master/swift/Dockerfile was a good example of this too, thanks for sending it over.

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile (hunk 2 above)
Comment (mcdonamp) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: Done

File: tutorials/run-swift-vapor-on-google-cloud/Dockerfile (hunk 1 above)
Comment (mcdonamp) [inline tutorials/run-swift-vapor-on-google-cloud/Dockerfile]: Done

**Pull request merged**